### PR TITLE
compute: save args of qemu to file

### PR DESCRIPTION
**Pull request by weia5-emc**
save args to file.

### CODE REVIEW

**Comment by weia5-emc** [comment]
retest this please

File: test/functional/test_node_ports.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ def test_start_node_with_conflict_bmc_connection_port(self):
             node2 = model.CNode(self.node_info_2)
             node2.init()
             node2.precheck()
-        except ArgsNotCorrect, e:
+        except ArgsNotCorrect as e:
```
Comment (XiaowenJiang) [inline test/functional/test_node_ports.py]: what's the difference between these two usages?

File: test/functional/test_storage.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def test_controller_with_drive2(self):
             "max_sge": 128,
             "max_drive_per_controller": 6,
             "drives": drive2
-            }]
+        }]
```
Comment (XiaowenJiang) [inline test/functional/test_storage.py]: cool

File: test/functional/test_storage.py [hunk 3]
```diff
@@ -191,20 +193,20 @@ def test_controller_with_drive2(self):
...
                                                            "max_drive_per_controller": 6,
                                                            "drives": [
-                                                                {
-                                                                    'file': fixtures.image,
-                                                                    'bootindex': 1,
-                                                                    'use_msi': 'true',
-                                                                    'size': 8
-                                                                }
-                                                            ]
+                                                               {
```
Comment (XiaowenJiang) [inline test/functional/test_storage.py]: good catch

File: test/functional/test_node_ports.py (hunk 1 above)
Comment (weia5-emc) [reply test/functional/test_node_ports.py]: "ArgsNotCorrect, e" is not standard style, which is not support by python 3.

File: test/functional/test_storage.py (hunk 3 above)
Comment (weia5-emc) [reply test/functional/test_storage.py]: IDE performs style reformation automatically for me, not me.

**Pull request merged**